frontend/codegen.ts

```diff
@@ -0,0 +1,21 @@
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: 'http://localhost:8080/graphql',
+  documents: 'src/**/*.ts',
```

Comment:
```suggestion
  documents: 'src/graphql/**/*.ts',
```
Assim ele não precisa procurar em todos os arquivos

Reply:
👆🏾